src/pgduckdb_types.cpp

```diff
@@ -543,8 +543,12 @@ ConvertPostgresToDuckColumnType(Form_pg_attribute &attribute) {
 		}
 		return duck_type;
 	}
-	default:
-		return duckdb::LogicalType::USER("UnsupportedPostgresType");
+	case REGCLASSOID:
+		return duckdb::LogicalTypeId::INTEGER;
```

Comment:
That works, do keep in mind that this means we lose the context of what this was on the postgres side this way.
We'd need something similar to what we do for `NumericAsDouble` to differentiate between an actual INT4 and a REGCLASSOID